mdsuite/database/simulation_database.py

```diff
@@ -194,7 +198,11 @@ def add_data(self, data: np.array, structure: dict, start_index: int, batch_size
                     database[item][start_index:stop_index, :] = data[structure[item]['indices']][
                         np.s_[:, structure[item]['columns'][0]:structure[item]['columns'][-1] + 1]].astype(float)
                 else:
-                    database[item][:, start_index:stop_index, :] = self._get_data(data, structure, item, batch_size, sort,
+                    database[item][:, start_index:stop_index, :] = self._get_data(data,
+                                                                                  structure,
+                                                                                  item,
+                                                                                  batch_size,
+                                                                                  sort,
                                                                                   n_atoms=n_atoms)
```

Comment:
Don't know about that, I personaly would go for something like this:
```py
database[item][:, start_index:stop_index, :] = self._get_data(
    data, structure, item, batch_size, sort, n_atoms=n_atoms
)
```
Was this something black reformatted?